src/controller/get.controller.ts

```diff
@@ -34,16 +36,26 @@ export class GetController {
   ): Promise<void> {
     try {
       if (domains.length === 0) {
-        throw new Error(
-          'Please provide a domain (functions, deployments and/or account)',
-        );
+        const error: PrettyPrintableError = {
+          message:
+            'Please provide a domain (functions, deployments and/or account)',
+          suggestions: ['Functions, deployments, account or events.'],
+          ref: 'https://github.com/LivePersonInc/faas-cli#get',
+          code: CLIErrorCodes.DomainMissing,
+        };
+        throw error;
```

Comment:
If you throw the error that way, you have to remove the local `catch` at the end of the file. Otherwise, the error will be forwarded to the view and not to the `command`.